scripts/run-e2e.ts

```diff
@@ -123,120 +114,202 @@ const runTests = async ({ name, ...rest }: Parameters) => {
+    logger.info(`📑 Running test for ALL frameworks`);
+    Object.values(typedConfigs).forEach((config) => {
+      e2eConfigs[`${config.name}-${config.version}`] = config;
+    });
+  }
+
+  // Compute the list of frameworks we will run E2E for
+  if (frameworkArgs.length > 0) {
+    frameworkArgs.forEach((framework) => {
+      e2eConfigs[framework] = Object.values(typedConfigs).find((c) => c.name === framework);
+    });
+  } else {
+    const selectedValues = await prompts([
+      {
+        type: 'toggle',
+        name: 'openCypressInUIMode',
+        message: 'Open cypress in UI mode',
+        initial: false,
+        active: 'yes',
+        inactive: 'no',
+      },
+      {
+        type: 'toggle',
+        name: 'useLocalSbCli',
+        message: 'Use local Storybook CLI',
+        initial: false,
+        active: 'yes',
+        inactive: 'no',
+      },
+      {
+        type: 'autocompleteMultiselect',
+        message: 'Select the frameworks to run',
+        name: 'frameworks',
+        hint:
+          'You can also run directly with package name like `test:e2e-framework react`, or `yarn test:e2e-framework --all` for all packages!',
+        choices: Object.keys(configs).map((key) => {
+          // @ts-ignore
+          const { name, version } = configs[key];
+          return {
+            // @ts-ignore
+            value: configs[key],
+            title: `${name}@${version}`,
+            selected: false,
+          };
+        }),
+      },
+    ]);
+
+    if (!selectedValues.frameworks) {
+      logger.info(`No framework was selected.`);
+      process.exit(process.exitCode || 0);
+    }
+
+    useLocalSbCli = selectedValues.useLocalSbCli;
+    openCypressInUIMode = selectedValues.openCypressInUIMode;
+    e2eConfigs = selectedValues.frameworks;
+  }
 
-const perform = () => {
-  const limit = pLimit(1);
+  // Remove frameworks listed with `--skip` arg
+  frameworksToSkip.forEach((framework) => {
+    delete e2eConfigs[framework];
+  });
+};
+
+const perform = async (): Promise<Record<string, boolean>> => {
+  await getConfig();
   const narrowedConfigs = Object.values(e2eConfigs);
 
   const list = filterDataForCurrentCircleCINode(narrowedConfigs) as Parameters[];
 
   logger.info(`📑 Will run E2E tests for:${list.map((c) => `${c.name}`).join(', ')}`);
 
-  return Promise.all(list.map((config) => limit(() => runE2E(config))));
+  const e2eResult: Record<string, boolean> = {};
+
+  // Run all e2e tests one after another and fill result map
+  await list.reduce(
+    (previousValue, config) =>
+      previousValue
+        .then(() => runE2E(config))
+        .then((result) => {
+          e2eResult[config.name] = result;
+        }),
+    Promise.resolve()
+  );
+
+  return e2eResult;
 };
 
-perform().then(() => {
+perform().then((e2eResult) => {
+  logger.info(`🧮 E2E Results`);
+
+  Object.entries(e2eResult).forEach(([configName, result]) => {
+    logger.info(`${configName}: ${result ? 'OK' : 'KO'}`);
```

Comment:
This is nice! But KO reads very similar to OK, maybe another word would be better?